scripts/wrappers/join.py

```diff
@@ -728,8 +727,6 @@ def update_dqlite(cluster_cert, cluster_key, voters, host):
             waits -= 1
     print(" ")
 
-    restart_all_services()
```

Comment:
I think that if we want to remove this line we should start/restart the API server after the k8s-dqlite service is up, or else we risk the API server timing out on k8s-dqlite and not starting at all. I guess it is a race condition we will not see often.